R/bayesmixr/R/zzz.R

```diff
@@ -0,0 +1,10 @@
+# Parse internal renviron file to set BAYESMIX_EXE variable
```

Comment:
This is a funny filename!

Reply:
Weird name for me, but this is common use. Quoting from [here](https://r-pkgs.org/r.html#when-you-do-need-side-effects):
> By convention, [.onLoad()](https://rdrr.io/r/base/ns-hooks.html) and friends are usually saved in a file called `R/zzz.R`